recipes/setuptools-git/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{%set name = "setuptools-git" %}
+{%set version = "1.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "047d7595546635edebef226bc566579d422ccc48a8a91c7d32d8bd174f68f831" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  preserve_egg_dir: True
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
```

Comment:
Based on the name of the package I am guessing that `setuptools` **is** a run time dependency.